doc/formatting.rst

```diff
@@ -18,8 +18,9 @@ Furthermore it can be used to format code of any defined dialect (see
 Configuring automatic formatting (dune 2.0)
 ===========================================
 
-If using ``(lang dune 2.0)``, there is nothing to do, formatting will be set up
-by default.
+If using ``(lang dune 2.0)``, there is nothing to setup in dune, formatting will
+be set up by default. However, ocamlformat will still refuse to format sources
```

Comment:
```suggestion
be set up by default. However, ``ocamlformat`` will still refuse to format sources
```

Reply:
Actually, I think that it would be `ocamlformat_`. It's a reference, rather than code.

Reply:
Indeed, thanks.